sympy/concrete/summations.py

```diff
@@ -517,11 +517,12 @@ def is_convergent(self):
         ratio = combsimp(powsimp(next_sequence_term/sequence_term))
         try:
             lim_ratio = limit_seq(ratio, sym)
-            if lim_ratio is not None and lim_ratio.is_number:
-                if abs(lim_ratio) > 1:
-                    return S.false
-                if abs(lim_ratio) < 1:
-                    return S.true
+            if not isinstance(lim_ratio, type(S.NaN)):
```

Comment:
You can merge this with the below `if` by adding `and lim_ratio is not S.NaN`

Reply:
I have made appropriate changes ,hence resolving